app/subscriber/src/components/table/styled/SubscriberTableContainer.tsx

```diff
@@ -48,6 +48,12 @@ export const SubscriberTableContainer = styled(Col)`
 
     background: ${(props) => props.theme.css.bkWhite};
 
+    // rather than updating the tno-core package and adding props to control the zebra striping of the table with more props, we'll just override the css here
+    // as this wrapper will be applied to most tables in the subscriber app.
+    &:nth-child(2n) {
+      background: inherit !important;
```

Comment:
ugh, why not upgrade tno-core?

Reply:
Lemme do that, just wasn't sure if it was worth a prop or not :laughing:

Reply:
Updated, no more `!important`